render.c

```diff
@@ -40,6 +41,17 @@ void render(struct slurp_output *output) {
 		int x, y, width, height;
 		pointer_get_box(pointer, &x, &y, &width, &height);
 
+		if (display_dimensions) {
+			cairo_select_font_face(cairo, "Sans", CAIRO_FONT_SLANT_NORMAL,
+				CAIRO_FONT_WEIGHT_NORMAL);
+			cairo_set_font_size(cairo, 14);
+			// buffer of 11 can hold selections up to 99999x99999
+			char dimensions[11];
+			sprintf(dimensions, "%ix%i", width, height);
+			cairo_move_to(cairo, x + width + 15, y + height + 25);
+			cairo_show_text(cairo, dimensions);
```

Comment:
Hmm, I'm afraid we're going to need more elaborate stuff for HiDPI. Probably with `pango_attr_list_insert(attrs, pango_attr_scale_new(scale));`, see https://github.com/emersion/mako/blob/master/render.c#L76 (though we don't need the pango markup stuff).

To test HiDPI you can compile wlroots, configure the `WL-1` output with `scale=2`, run rootston inside another Wayland/X11 session and start slurp from within rootston. Not that easy, but doable ;)